.github/workflows/main.yaml

```diff
@@ -66,19 +68,10 @@ jobs:
       - name: Setup NodeJS
         uses: ./.github/workflows/actions/setup-node
 
-      - name: Setup Java
```

Comment:
Why are we getting rid of this step? I thought it was recently added by @cvarjao

Reply:
ya, iOS does not use Java for anything so its unnecessary.

Reply:
if we don't need it, I am fine with taking it out